nservicebus/sagas/index.md

```diff
@@ -22,6 +22,24 @@ Design processes with more than one remote call to use sagas.
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
 
+## No saga instance available
```

Comment:
We need a better title, it's not very descriptive, e.g. "Saga design considerations"?

Reply:
Maybe call this "What happens when a saga does not exist?"

Reply:
@timbussmann @weralabaj Below the note, it primarily is about out of order delivery/processing. The note states always assume out of order delivery. How to introduce the user to this?

Reply:
I'd opt for the generic title, like "Saga design considerations" because if it's too narrow then the reader is more likely to assume it's not relevant to them (I think). From the content it's very clear what the problem is, you repeat a few times "not found" and "out of order", so search should be fine too.

> How to introduce the user to this?

I think the introduction is fine, just it looks like you talk about something, part 1, then is the note, and then you continue talking about the same thing, part 2, but the connection between part 1 and 2 is not very clear. At least for me it sounds like part 2 is again starting from scratch, which is a bit confusing and that's why I'm not sure if I got the intention right.

If you say, yes, I'm talking about the same thing, and there's a note in the middle of it, then I'm fine, it's all clear ;)

Reply:
@weralabaj Well, they are two different sections. The first mentioning the saga not found discard behavior, the second that this can have implications if you assume ordered delivery but in reality it is not and how to deal with out or order delivery. Maybe adding headings makes it all more clear.